rcl_action/CMakeLists.txt

```diff
@@ -86,6 +94,34 @@ if(BUILD_TESTING)
     )
     ament_target_dependencies(test_action_client "rcl" "test_msgs")
   endif()
+  ament_add_gtest(test_action_communication
+    test/rcl_action/test_action_communication.cpp
+  )
```

Comment:
This test (as well as the one below) is only covering the default rmw impl. It would be good if they test would be invoked for every rmw impl.